src/main/java/net/sf/mpxj/mpp/Var2Data.java

```diff
@@ -379,6 +379,36 @@ public long getLong(Integer id, Integer type)
       return (result);
    }
 
+   /**
+    * This method retrieves a long integer of the specified type,
+    * belonging to the item with the specified unique ID. Note that
+    * the long value is read from an arbitrary offset within the
+    * byte array of data.
+    *
+    * @param id unique ID of entity to which this data belongs
+    * @param offset offset into the byte array fom which to read the long integer
+    * @param type data type identifier
+    * @return required long integer data
+    */
+   public long getLong(Integer id, int offset, Integer type)
```

Comment:
This method is actually unused now.  I created it when trying to build the GUID but switched to using the existing helper method in MPPUtility when I saw the GUID vs. UUID differences.